src/components/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
i would prefer if you use single quotes for js code and double for html code, is normal use that practice :)